MMM-COVID19.js

```diff
@@ -180,47 +198,57 @@ Module.register("MMM-COVID19", {
         let countryRow = document.createElement("tr"),
             countryNameCell = document.createElement("td"),
             confirmedCell = document.createElement("td"),
+            casesPerMCell = document.createElement("td"),
             newCasesCell = document.createElement("td"),
             deathsCell = document.createElement("td"),
             newDeathsCell = document.createElement("td"),
+            seriousCell = document.createElement("td"),
             recoveredCell = document.createElement("td"),
             activeCell = document.createElement("td"),
             countryName = value["country_name"],
             cases = value["cases"],
             deaths = value["deaths"],
+	    serious = value["serious_critical"],
             newCases = value["new_cases"],
             newDeaths = value["new_deaths"],
             totalRecovered = value["total_recovered"],
-            activeCases = value["active_cases"];
+            activeCases = value["active_cases"],
+	    casesPerM = value["total_cases_per_1m_population"];
 
         countryNameCell.innerHTML = countryName
         countryNameCell.className = this.config.infoRowClass
         confirmedCell.className = 'number confirmed ' + this.config.infoRowClass
         confirmedCell.innerHTML = cases
+        casesPerMCell.className = 'number cases per M ' + this.config.infoRowClass
+        casesPerMCell.innerHTML = casesPerM
         newCasesCell.className = 'number confirmed ' + this.config.infoRowClass
         if (newCases) {
-          newCasesCell.innerHTML = '+' + newCases
+          newCasesCell.innerHTML = newCases
         }
         deathsCell.className = 'number deaths ' + this.config.infoRowClass
         deathsCell.innerHTML = deaths
         newDeathsCell.className = 'number deaths ' + this.config.infoRowClass
         if (newDeaths) {
-          newDeathsCell.innerHTML = '+' + newDeaths
+          newDeathsCell.innerHTML = newDeaths
```

Comment:
@testdruif keep this as it was (nothing to do with this PR)

Reply:
Bit strange to add a + since it will always be positive.
Adding the + kind off disturbs the view of the table because it's only used there.